server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowTaskInstanceService.java

```diff
@@ -255,6 +256,8 @@ public List<? extends FlowTaskResult> getTaskResultFromEntity(@NotNull TaskEntit
             return getApplyProjectResult(taskEntity);
         } else if (taskEntity.getTaskType() == TaskType.APPLY_DATABASE_PERMISSION) {
             return getApplyDatabaseResult(taskEntity);
+        } else if (taskEntity.getTaskType() == TaskType.APPLY_TABLE_PERMISSION) {
+            return getApplyTableResult(taskEntity);
```

Comment:
Apply table permission task has no file for download, so please add `instance.getTaskType() != TaskType.APPLY_TABLE_PERMISSION` in `FlowTaskInstanceService#getDownloadableTaskEntity()`.